dev/TabView/TabView.idl

```diff
@@ -1,67 +1,89 @@
 ﻿namespace MU_XC_NAMESPACE
 {
 
-[WUXC_VERSION_PREVIEW]
+[WUXC_VERSION_MUXONLY]
 [webhosthidden]
 enum TabViewWidthMode
 {
     SizeToContent = 0,
```

Comment:
The zero value should be Equal now, since it's the default value.

Reply:
Fixed!